dispatcher/src/test/java/io/zeebe/dispatcher/DispatcherTest.java

```diff
@@ -89,12 +89,15 @@ public void setup() {
             publisherLimit,
             publisherPosition,
             A_LOG_WINDOW_LENGTH,
```

Comment:
The creation of the dispatcher it totally unclear - so many parameters :see_no_evil:

Reply:
True. I hope we can get rid of it entirely. So, I don't want to invest here too much.

Reply:
Agree